cla_backend/apps/reports/urls.py

```diff
@@ -40,6 +40,11 @@
     url(r"^mi-eod-extract/$", views.mi_eod_extract, name="mi_eod_extract"),
     url(r"^mi-comlpaints/$", views.mi_complaints, name="mi_complaints"),
     url(r"^mi-obiee-extract/$", views.mi_obiee_extract, name="mi_obiee_extract"),
+    url(
+        r"^mi-tell-us-about-your-problem/$",
+        views.tell_us_more_about_your_problem,
+        name="mi_tell_us_about_your_problem",
```

Comment:
Is the title of this report intended to be 'Tell us more about your problem'? 
In the [ticket](https://dsdmoj.atlassian.net/jira/software/c/projects/LGA/boards/226?selectedIssue=LGA-2984) this report was given the title 'Problem categorisation'